src/toggle/icon.d.ts

```diff
@@ -0,0 +1,70 @@
+interface IconInterface {
```

Comment:
why is this in toggle folder? shouldn't be in inside icon folder? Also does icons work? as we only do index.jsx as entry